### DISCUSSION TITLE
编辑单元格时，如果没有失焦，就没法拿到修改后的值吗？目前save()方法拿不到

### DISCUSSION THREAD

**Discussion by 352836670** [dream-num/univer#2290] [Q&A | 常见问题]
编辑单元格时，如果没有失焦，就没法拿到修改后的值吗？目前save()方法拿不到

**Comment by 352836670** [comment]
而且单元格失去焦点，只能点击另一个单元格，点univer实例之外的区域，没法让单元格失去焦点吗？

**Comment by jikkai** [comment]
https://github.com/dream-num/univer/issues/1314#issuecomment-2027192535
